Dev/Install/config-form.html

```diff
@@ -6,7 +6,7 @@
 	<head>
 		<title>SilverStripe CMS / Framework Installation</title>
 		<meta http-equiv="Content-type" content="text/html; charset=utf-8">
-		<script type="application/javascript" src="<?php echo FRAMEWORK_NAME; ?>/thirdparty/jquery/jquery.js"></script>
+		<script type="application/javascript" src="<?php echo FRAMEWORK_NAME; ?>/admin/thirdparty/jquery/jquery.js"></script>
```

Comment:
Best bet is to refer to a CDN copy of jQuery: `http://code.jquery.com/jquery-1.7.2.min.js`. Otherwise this will break when we split out admin. It's also an inappropriate coupling of the installer to the admin.